src/js/providers/providers.js

```diff
@@ -58,9 +58,11 @@ define([
             });
         }
 
-        var F = function(){};
-        F.prototype = Default;
-        provider.prototype = new F();
+        if (name !== 'hlsjs') {
+            var F = function () {};
+            F.prototype = Default;
+            provider.prototype = new F();
+        }
```

Comment:
There are issues removing it so how about this?

`_.defaults(provider.prototype, Default);`